apps/backend/src/plans/fiches/list-fiches/list-fiches.service.ts

```diff
@@ -1006,6 +1098,159 @@ export default class ListFichesService {
     }
   }
 
+  private getHasNoPlanCondition(
+    noPlan: boolean | undefined,
+    collectiviteId: number | null
+  ): SQLWrapper | undefined {
+    if (!isNil(noPlan)) {
```

Comment:
détail mais return early avec condition positive possible ici

```suggestion
if (isNil(noPlan)) return;